cypress/integration/save.cy.js

```diff
@@ -184,21 +185,36 @@ describe('saving an AO', () => {
         })
     })
 
-    /*
-    describe('"save" for a saved AO created by others', () => {
-        it('navigates to the start page', () => {
+    // Specific test for DHIS2-15722
+    describe('"save" a copied AO created by others', () => {
+        const TEST_VIS_BY_OTHERS_NAME = 'ANC: 1-3 dropout rate Yearly'
+        const TEST_VIS_BY_OTHERS_NAME_UPDATED = `${TEST_VIS_BY_OTHERS_NAME} - updated`
+
+        it('navigates to the start page and opens a random AO created by others', () => {
             goToStartPage()
+            openAOByName(TEST_VIS_BY_OTHERS_NAME)
+            expectAOTitleToBeValue(TEST_VIS_BY_OTHERS_NAME)
         })
-        it('opens a random AO created by others', () => {
-            openRandomAOCreatedByOthers()
+        it('saves AO using "Save As"', () => {
```

Comment:
This is using the old format (found throughout DV) with multiple `it`. The "proper" format is to have one `it` per specific case and use `//` comments or line breaks to separate different sections of the case when needed.
I.e. since this is just a single test case, remove all but the first `it` (and change the title accordingly).